test/integration/client-side-encryption/client_side_encryption.spec.test.ts

```diff
@@ -75,7 +75,7 @@ describe('Client Side Encryption (Legacy)', function () {
   const testSuites = gatherTestSuites(
     path.join(__dirname, '../../spec/client-side-encryption/tests/legacy'),
     testContext
-  );
+  ).filter(({ name }) => !name.includes('fle2-Range'));
```

Comment:
Not our usual method for skipping tests because they do not appear in test output - but because I am about to implement the logic for these tests, it was easier than listing out all tests to skip into an array.